rclpy/src/rclpy/_rclpy.c

```diff
@@ -2979,14 +2979,15 @@ rclpy_get_service_names_and_types_by_node(PyObject * Py_UNUSED(self), PyObject *
   return pyservice_names_and_types;
 }
 
-/// Get the list of published topics discovered by the provided node for the remote node name
+/// Get the list of subscription topics discovered by the provided node for the remote node name
```

Comment:
I would say `Get the list of subscribed topics`, though the whole brief is confusing. It really is doing something like:

> Get the list of topic names and types for which there is at least one subscription associated with the given node name.

But updating this one would mean updating the publisher version as well and may be out of scope for this pr.

Reply:
Clarified briefs 935c054